arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -805,6 +806,25 @@ when disk size is very constrained and no replication is used.)");
               arangodb::options::Flags::OnSingle))
       .setIntroducedIn(31005);
 
+  options
+      ->addOption("--rocksdb.force-legacy-little-endian-keys",
+                  "If enabled and a new RocksDB database is generated, "
+                  "the old legacy little endian key encoding will be "
+                  "used. This is bad for performance and disables a "
+                  "few features like parallel index generation and is "
+                  "thus only sensible for testing purposes. Don't use!",
```

Comment:
This description is repeated in the long description below. I suggest shortening this description to
```suggestion
                  "Force usage of legacy little endian key encoding when creating "
                  "a new RocksDB database directory. DO NOT USE IN PRODUCTION.",
```

Reply:
In hindsight, a better name for `.setLongDescription()` would be `.setDetails()` I suppose. Should we change this to avoid misunderstandings?

Reply:
Would be OK for me, but it does not seem to be urgent to me. What difference in meaning would be from `LongDescription` to `Details`?

Reply:
Description changed.